### DISCUSSION TITLE
Why when I try to set a task with blank status to success I get this message.

### DISCUSSION THREAD

**Discussion by sonnh97** [apache/airflow#28792] [Q&A]
url: http://ip:8080/api/v1/dags/raw_fds_daily/dagRuns/updateTaskInstancesState
body: {  
  "dry_run": false,
  "task_id": "check_source_status",
  "execution_date": "2022-09-10T18:00:00+00:00",
  "include_upstream": false,
  "include_downstream": false,
  "include_future": false,
  "include_past": false,
  "new_state": "success"
}

Result:
{
    "detail": "The method is not allowed for the requested URL.",
    "status": 405,
    "title": "Method Not Allowed",
    "type": "about:blank"
}

**Comment by Taragolis** [comment]
[Method Not Allowed](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#section/Errors/MethodNotAllowed) error raises when user try to use unsupported [HTTP methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) for specific API call, most probably you try to use one of GET/PUT/PATCH/DELETE method instead of POST which expected for [dags/{dag_id}/updateTaskInstancesState  ](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/post_set_task_instances_state)

**Reply by sonnh97** [reply]
I do not think so. Since I'm passing the tasks to mark success one by one, until one has an blank state. And i use post

**Reply by Taragolis** [reply]
If you think that something another happened rather than wrong HTTP Method, then you need spend some time and provide more detail information:
1. How do you interact with REST API: `curl`, `requests`, swagger, [airflow-client-python](https://github.com/apache/airflow-client-python), Postman and etc. 
2. Which version of Airflow are you use?
3. What kind of deployment.
4. Is airflow behind of some kind of proxy?
5. Some additional information

This info might help (but there is no guarantee) to give better advice rather than generic one.
